src/test/java/spiderman/DeadlineTest.java

```diff
@@ -0,0 +1,35 @@
+package spiderman;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I think static imports could be before other non-static import statements (in accordance with the coding standard).